src/listfunc.c

```diff
@@ -1778,8 +1839,11 @@ static StructGVarFunc GVarFuncs [] = {
     GVAR_FUNC(OnRight, 2, "pnt, elm"),
     GVAR_FUNC(OnLeftAntiOperation, 2, "pnt, elm"),
     GVAR_FUNC(OnLeftInverse, 2, "pnt, elm"),
-    GVAR_FUNC(COPY_LIST_ENTRIES, -1, "srclist,srcstart,srcinc,dstlist,dststart,dstinc,number"),
+    GVAR_FUNC(COPY_LIST_ENTRIES,
+              -1,
+              "srclist,srcstart,srcinc,dstlist,dststart,dstinc,number"),
```

Comment:
I assume `clang-format` produced this weird result? But why the `-1` on a separate line?!

Reply:
I presume it's clang-format - this diff was present on your original commit https://github.com/gap-system/gap/commit/0eb335dcfaeab1c2b5959e99f732caee50904e83

I'll put these lines back to how they were.